cudl-staging/terraform.tfvars

```diff
@@ -186,13 +189,26 @@ db-lambda-information = [
     "description"   = "Transforms the UI json file into a json format with suitable paths for the viewer / db"
     "jar_path"      = "release/uk/ac/cam/lib/cudl/awslambda/AWSLambda_Data_Transform/0.16/AWSLambda_Data_Transform-0.16-jar-with-dependencies.jar"
     "queue_name"    = "CUDLPackageDataUIQueue"
+    "transcription" = false
     "timeout"       = 900
     "memory"        = 512
-    "filter_prefix" = "cudl.ui.json"
     "handler"       = "uk.ac.cam.lib.cudl.awslambda.handlers.UIFileDBHandler::handleRequest"
     "runtime"       = "java11"
   }
 ]
+enhancements-lambda-information = [{
+  "name"          = "AWSLambda_CUDLDataEnhancements_TranskribusMergeTEI"
+  "description"   = "Used by the Transkribus pipeline to merge TEI transcription output from Transkribus into the TEI CUDL metadata.  Enhances the CUDL TEI with the Transkribus transcription data"
+  "jar_path"      = "release/uk/ac/cam/lib/cudl/awslambda/AWSLambda_Data_Transform/0.16/AWSLambda_Data_Transform-0.16-jar-with-dependencies.jar"
+  "queue_name"    = "CUDLTranskribusQueue"
+  "transcription" = true
+  "timeout"       = 900
+  "memory"        = 512
+  "handler"       = "uk.ac.cam.lib.cudl.awslambda.handlers.XSLTTransformRequestHandler::handleRequest"
+  "runtime"       = "java11"
+}]
+db-lambda-information = [
+]
```

Comment:
Rather than passing in an empty list here, I think it would make sense to remove the variable `db-lambda-information` from the `cudl-data-processing` module as It is no longer used

Reply:
Removed.